data/EngConf.txt

```diff
@@ -0,0 +1,808 @@
+# This dataset is licensed under a Creative Commons Attribution 4.0 International License. https://creativecommons.org/licenses/by/4.0/
```

Comment:
could you add a README to the "data" folder with a brief description of the datasets, links to the original sources, and usage?